README.md

```diff
@@ -41,7 +41,7 @@ Use your key and secret credentials; Find your proper region in aws console (def
 
 
 # Turn on RASA server
-EC2 server can be configured to on and off automatically, Currently turns on every sunday from 1 to 5
+EC2 server can be configured to on and off automatically, Currently turns on every Sunday from 1 to 5
```

Comment:
Good catch, what about "Currently"? Uncapitalize this.

"EC2 server can be configured to on and off automatically"<- I think the grammar here is off

Reply:
I'll make those edits. Thanks.

Reply:
Question: is it 1am to 5am or 1pm to 5pm?

Reply:
ah yes, afternoon. 1pm to 5pm

Reply:
Great job! Resolving and merging.